### ISSUE TITLE
Links in bottom navigation some duplicates, non functioning

### ISSUE THREAD

**Issue by thunder-** [interchange/Interchange6-Demo-Shop#29]
This is the list of topics in the bottom navigation:
    Contact us
    Privacy policy
    Terms and conditions
    Return orders
    Gift certificates
    Contact us
    Contact us
    Contact us

4x Contact us, none of the links work. Either 404 or refreshes the current page.

**Comment by SysPete** [comment]
Although all are still 404 the bottom navigation is now built from Navigation entries in the database so has no duplicates.

**Comment by racke** [comment]
The "legal" stuff could be covered by entries in the message table, so it is easy to modify in the database.
What do you think?

**Comment by SysPete** [comment]
Yup - we should probably create these in Navigation with template set to 'message' then if we see that template we use just grab content (markdown?) and throw at template using MD filter.
